src/swarm/neo/protocol/socket/MessageReceiver.d

```diff
@@ -383,17 +383,15 @@ private class MessageReceiverBase
     ***************************************************************************/
 
     private void ensureMinimumAmountOfBytesInBuffer ( size_t bytes_requested, lazy Event wait )
-    in
-    {
-        assert(this); // invariant
-    }
     out
     {
         assert(bytes_requested <= this.buffer_content_end);
         assert(this); // invariant
     }
     body
     {
+        verify(this !is null);
```

Comment:
This is wrong. Doing `assert(this);` causes the invariant to be called. (Hence the comment.)

Reply:
Would be nice to not make this intermediary wrong change here.